Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/server/VerificationServer.kt

```diff
@@ -51,7 +52,7 @@ class VerificationServer @Inject constructor(
 
     suspend fun pollTestResult(
         token: RegistrationToken
-    ): CoronaTestResult = withContext(Dispatchers.IO) {
+    ): Pair<CoronaTestResult, Instant?> = withContext(Dispatchers.IO) {
```

Comment:
Cn you create a data class for that ?

Reply:
Maybe 
```kotlin
data CoronaTestResultResponse(val testResult:Result, val timesamp:Int?)
```

So when it is used down below, it is clear which is which  instead of `first` and `second `